spk/homeassistant/src/installer.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python3"
+PATH="${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+GROUP="users"
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+CONFIG_DIR="${INSTALL_DIR}/etc"
```

Comment:
Why? It's nothing special :) If you're worried it'll be backed up when you upgrade: it won't, because when the package is stopped, the PID file is already removed.